cirq-google/cirq_google/api/v2/run_context.proto

```diff
@@ -225,9 +225,13 @@ message SingleSweep {
 
 // A list of explicit values.
 message Points {
-  // The values.
+  // The values. (it is deprecated because it is float32.)
   repeated float points = 1;
 
+  // Due to historical reason, the points are stored as float32.
+  // But float64 should be a better choice.
+  repeated double points_fl64 = 3;
```

Comment:
Nit - consider renaming to `points_double` here and similarly in `Linspace` for consistency with `ConstValue.double_value`.

Reply:
ok